project_task_employee/tests/test_project_task.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2018 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests.common import TransactionCase
+
+
+class TestProjectTask(TransactionCase):
+    def setUp(self):
+        super(TestProjectTask, self).setUp()
+
+        self.task_3 = self.env['project.task'].browse(
```

Comment:
@ernestotejeda IMO Add one variable and use this obj `self.env['project.task']`.

Ex: self.project_task_obj =  self.env['project.task']